apps/web/src/components/Tabs.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import { useState } from "react";
+import clsx from "clsx";
```

Comment:
can we use `cn` from utils that combines `clsx` and `tailwind-merge`, apply this throughout, thanks!